src/Game.h

```diff
@@ -170,7 +170,7 @@ class Game {
 
 	std::unique_ptr<Player> m_player;
 
-	enum State {
+	enum class State {
 		STATE_NORMAL,
 		STATE_HYPERSPACE,
```

Comment:
I don't know what the naming convention is, but perhaps the variable don't need "STATE" prefix when it's given by the class, e.g. `state::STATE_NORMAL`?

Reply:
Good point I'll change that!